src/Pos/Wallet/SscType.hs

```diff
@@ -5,7 +5,8 @@ module Pos.Wallet.SscType
        ( WalletSscType
        ) where
 
-import           Pos.Ssc.GodTossing (SscGodTossing)
+import           Pos.Security.Workers ()
```

Comment:
It imports `instance SecurityWorkersClass SscGodTossing` I guess.

`Pos.Security` is weird.